pkg/transforms/outputdata_test.go

```diff
@@ -96,9 +95,7 @@ func TestSetOutputDataMultipleParametersValid(t *testing.T) {
 func TestSetOutputDataBadType(t *testing.T) {
 	target := NewOutputData()
 
-	data := startup.Endpoint{}
-
-	continuePipeline, result := target.SetOutputData(context, data)
+	continuePipeline, result := target.SetOutputData(context, make(chan int))
```

Comment:
Please comment why using a channel.

Reply:
Channels are not marshalable to JSON and are the standard way to generate that error.

Reply:
Sorry, I meant please add comment to code. What you responded with is perfect.

Reply:
This applies to similar comments ;-)